lib/libmbedtls/core/tee_lmd_provider.c

```diff
@@ -2127,6 +2259,13 @@ TEE_Result crypto_cipher_update(void *ctx, uint32_t algo,
 	int lmd_res;
 	size_t olen;
 	size_t finish_olen;
+#if defined(CFG_CRYPTO_CTS)
+	struct tee_symmetric_cts *cts;
```

Comment:
I'd prefer if the `#if` was skipped and the `__maybe_unused` tag was used instead, here and the two other locals below.

Reply:
Sorry for just copy code to here directly. I will move the declare to the block where use it.